.gitignore

```diff
@@ -2,6 +2,7 @@
 /lib/
 /bin/
 /dist/
+/builddir/
```

Comment:
This is not required, BTW. :)

Meson will create `builddir/.gitignore` with the contents `*` specifically because it always knows the builddir and also knows that anything in it should not be checked into git.

On the other hand, if you *did* need to add it to gitignore yourself, it could have any arbitrary name, `builddir` is popular but not the only one people use.

Reply:
Oh nice catch!